jerry-port/default/default-io.c

```diff
@@ -98,3 +99,79 @@ jerry_port_log (jerry_log_level_t level, /**< message log level */
     va_end (args);
   }
 } /* jerry_port_log */
+
+#ifndef CONFIG_DISABLE_ES2015_MODULE_SYSTEM
+
+/**
+ * Determines the size of the given file.
+ * @return size of the file
+ */
+static size_t
+jerry_port_get_file_size (FILE *file_p) /**< opened file */
+{
+  fseek (file_p, 0, SEEK_END);
+  long size = ftell (file_p);
+  fseek (file_p, 0, SEEK_SET);
+
+  return (size_t) size;
+} /* jerry_port_get_file_size */
+
+/**
+ * Opens file with the given path and reads its source.
+ * @return the source of the file
+ */
+uint8_t *
+jerry_port_module_read_source (const char *file_name_p, /**< file name */
```

Comment:
The documentation update is missing for the new port API functions. BTW, why are these needed? Might be better to put the port API changes into a separate PR.

Reply:
Thanks for the review, I’ve updated the port API documentation. The port functions are necessary for the modules, because without these Jerry could not open custom files (source files). For this reason, the changes can’t be separated into another PR.